tests/Formatter/ClassPropertiesFormatterTest.php

```diff
@@ -16,47 +16,23 @@ class ClassPropertiesFormatterTest extends \PHPUnit_Framework_TestCase
     private $formatter;
 
     /**
-     * @var PropertySerializer|MockInterface
+     * @var PropertyNormalizer|MockInterface
      */
-    private $serializer;
+    private $normalizer;
 
     protected function setUp()
     {
-        $this->serializer = Mockery::mock(PropertySerializer::class);
-        $this->serializer->shouldReceive('encode')->andReturn('!!!');
+        $this->normalizer = Mockery::mock(PropertyNormalizer::class);
+        $this->normalizer->shouldReceive('normalize')->andReturn('!!!');
 
-        $this->formatter = new ClassPropertiesFormatter($this->serializer);
+        $this->formatter = new ClassPropertiesFormatter($this->normalizer);
     }
 
-    public function testCommandReceivedReturnsExpectedMessage()
+    public function testPropertiesUsedReturnsNormalizationResult()
     {
-        $command = new RegisterUserCommand();
-
-        $this->assertEquals(
-            'Command received: League\Tactician\Logger\Tests\Fixtures\RegisterUserCommand !!!',
-            $this->formatter->commandReceived($command)
-        );
-    }
-
-    public function testCommandHandledReturnsExpectedMessage()
-    {
-        $command = new RegisterUserCommand();
-
-        $this->assertEquals(
-            'Command succeeded: League\Tactician\Logger\Tests\Fixtures\RegisterUserCommand !!!',
-            $this->formatter->commandHandled($command)
-        );
-    }
```

Comment:
These removed tests are still valid. We should extend the ClassNameFormatter test to inherit them as well or duplicate them.